doc/api/dgram.md

```diff
@@ -1,5 +1,7 @@
 # UDP / Datagram Sockets
 
+<!--introduced_in=v0.10.0-->
```

Comment:
I don't think this one is correct, although I'm not sure we care since we are only going back to v0.10